app/data/events/correlation-id/get-events-by-correlation-id.js

```diff
@@ -0,0 +1,18 @@
+const { CORRELATION_ID } = require('../../../constants/categories')
+const { addPendingEvents } = require('../add-pending-events')
+const { getEvents } = require('../get-events')
+const { groupEventsByCorrelationId } = require('./group-events')
+const { orderGroupedEvents } = require('./order-grouped-events')
+const { sanitiseEvents } = require('../sanitise-events')
```

Comment:
Apart from the constant, maybe sort these in the order they appear below?

Reply:
I see that the file has been updated, but I don't notice any change in the order?

Reply:
Should be done now